### PR TITLE
Timp

### LINKED ISSUE #182: Timp Release
Ref: [Ops Order Fulfillment Refactor](https://dobadev.atlassian.net/wiki/spaces/TT/pages/334168080/Orders+Ops+Fullfillment+Refactor)

This release supports the 'timp' release.  The documentation site should support access to both the current release and the new 'timp' release with 'timp' being the default.  This might include creating a versioning document to allow users to navigate between releases.

#### Modify :
- [x] `POST: /timp/orders/create/` - Retailer only
- [x] `PUT: /timp/orders/allocation/{order_uuid}` - Supplier only
- [x] `POST: /timp/orders/tracking/{order_uuid}` - Supplier only
- [x] `POST: /timp/orders/{order_uuid}`
- [x] `POST: /timp/orders/`
- [x] `POST: /timp/orders/{order_uuid}/fees `

#### New :
- [x] `PUT: /timp/orders/tracking/{order_uuid}/{old_tracking_number}` - Supplier only
- [x] `POST: /timp/orders/{order_uuid}/attributes/ `
- [x] `GET: /timp/orders/{order_uuid}/attributes/ `
- [x] `POST: /timp/orders/{order_uuid}/invoices/` - Supplier only
- [x] `GET: /timp/orders/{order_uuid}/invoices/` - Retailer only
- [x] `PATCH: /timp/orders/cancel/{order_uuid/reject}/` - Supplier only
- [x] `PATCH: /timp/orders/cancel/{order_uuid}/confirm/ ` - Supplier only
- [x] `PATCH: /timp/orders/items/cancel/{order_uuid}/confirm/` - Supplier only
- [x] `PATCH: /timp/orders/items/cancel/{order_uuid}/reject/` - Supplier only

#### New
- [x] `PUT: /timp/orders/acknowledge` 

## Product Endpoints
For Get requests that return multiples (e.g. catalogs, items skus, inventory_lists), update the end-point to include a pagination object and a key for the array.  For example:
{ pagination: {...}, "skus": [ ...] }

- [x] `GET: /timp/products/catalogs/`
- [x] `GET: /timp/products/items/`
- [x] `GET: /timp/products/skus/`
- [x] `GET: /timp/products/inventory-lists/`

## Current Release Docs
We need to copy all the current release docs into _v1_<group> and then make them available by accessing the <server>/v1 url.  This will include making their visibility: 'v1'

The current docs (with the exception of those changed by Ops Order Fulfillment and Product refactors) will become part of the Timpanogas release.  We'll change the api endpoint in each current document to include `timp` in the path: `<server>/timp/<current api>`.

### PR DESCRIPTION
**Pull request by nickcoleman**
Update the docs to support the Timpanogas release

Ref: [Ops Order Fulfillment Refactor](https://dobadev.atlassian.net/wiki/spaces/TT/pages/334168080/Orders+Ops+Fullfillment+Refactor)

The documentation site should support access to both the current release and the new 'timp' release with 'timp' being the default.  This might include creating a versioning document to allow users to navigate between releases.

#### Modify :
- [x] `POST: /timp/orders/create/` - Retailer only
- [x] `PUT: /timp/orders/allocation/{order_uuid}` - Supplier only
- [x] `POST: /timp/orders/tracking/{order_uuid}` - Supplier only
- [x] `POST: /timp/orders/{order_uuid}`
- [x] `POST: /timp/orders/`
- [x] `POST: /timp/orders/{order_uuid}/fees `

#### New :
- [x] `PUT: /timp/orders/tracking/{order_uuid}/{old_tracking_number}` - Supplier only
- [x] `POST: /timp/orders/{order_uuid}/attributes/ `
- [x] `GET: /timp/orders/{order_uuid}/attributes/ `
- [x] `POST: /timp/orders/{order_uuid}/invoices/` - Supplier only
- [x] `GET: /timp/orders/{order_uuid}/invoices/` - Retailer only
- [x] `PATCH: /timp/orders/cancel/{order_uuid/reject}/` - Supplier only
- [x] `PATCH: /timp/orders/cancel/{order_uuid}/confirm/ ` - Supplier only
- [x] `PATCH: /timp/orders/items/cancel/{order_uuid}/confirm/` - Supplier only
- [x] `PATCH: /timp/orders/items/cancel/{order_uuid}/reject/` - Supplier only

#### New
- [x] `PUT: /timp/orders/acknowledge` 

## Product Endpoints
For Get requests that return multiples (e.g. catalogs, items skus, inventory_lists), update the end-point to include a pagination object and a key for the array.  For example:
{ pagination: {...}, "skus": [ ...] }

Note: these endpoints are based on discussion with @jtprince and are not active at time of this PR.

- [x] `GET: /timp/products/catalogs/`
- [x] `GET: /timp/products/items/`
- [x] `GET: /timp/products/skus/`
- [x] `GET: /timp/products/inventory-lists/`

## Current Release Docs
We need to copy all the current release docs into _v1_<group> and then make them available by accessing the <server>/v1 url.  This will include making their visibility: 'v1'

The current docs (with the exception of those changed by Ops Order Fulfillment and Product refactors) will become part of the Timpanogas release.  We'll change the api endpoint in each current document to include `timp` in the path: `<server>/timp/<current api>`.

Closes #182

### CODE REVIEW

File: _files/retrieve_export.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ right_code: |
 ---
 Retrieve a previously requested Export via the export_uuid. By calling this API Endpoint with the export_uuid in the URL, you can retrieve a previously requested Export.
 
+Note: this endpoint does not include 'timp' in the path
```
Comment (Gidgidonihah) [inline _files/retrieve_export.md]: Interesting. Do you know why this is?

File: _config.yml [hunk 2]
```diff
@@ -73,7 +69,7 @@ collections:  ### if there are no documents in a collection, it's a placeholder
     title: Operations Tools
     position: 8
     type: api
-###### Internal Collections - if there are no documents in a collection, it's a placeholder and visiblity is blank
+###### Timpanogas Internal Collections - if there are no documents in a collection, it's a placeholder and visiblity is blank
```
Comment (baileymatthewr) [inline _config.yml]: `Timpanogos` is the spelling

File: _documentation/api_version.md [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
 title: API Versions
-position: 1.1
+position: 2
 visibility: public
+---
+
+The most current version of the CruxConnect platform is named after the **Timpanogas** peak in the Wastach mountain range (and visible from our offices).
 
-To help you with your CruxConnect integration planing, we provide you access to both the current API endpoint (<a href="{{ site.baseurl}}">Current API</a>). And, to future changes to the API endpoints (<a href="{{ site.baseurl}}/future/">Future API</a>).
+The endpoints on this page document the Timpanogas release and start with `/timp/`
```
Comment (baileymatthewr) [inline _documentation/api_version.md]: `Timpanogos` is the spelling

**Comment by jtprince** [comment]
@nickcoleman - maybe I missed it (very possible), but I didn't see items nested under "items".  Did that get nested?

**Comment by baileymatthewr** [comment]
@nickcoleman : The "api-login" or whatever we're calling that endpoint these days doesn't get versioned. However, there is versioning that occurs on the related endpoints. For instance when you need to reset your password, you're going through a versioned endpoint. I don't know if this was on purpose or something else we should share with @jfischerlaunch & @jtprince ?

File: _files/retrieve_export.md (hunk 1 above)
Comment (nickcoleman) [reply _files/retrieve_export.md]: Not sure ... I can ping Jeff about it ... could have been an oversight

**Pull request merged**